lib/src/realm_class.dart

```diff
@@ -19,231 +19,79 @@
+    this._scheduler = _Scheduler(config);
+    this._realm = realmCore.openRealm(config);
   }
-  RealmObject _create(String? typeName, RealmObject object) native "Realm_create";
-
-  /// Gets all objects of type `T` 
-  /// 
-  /// Returns a [Results] object which can be indexed, filtered, sorted, queried etc
-  ///
-  /// ```dart
-  /// var objects = realm.objects<Car>();
-  /// Car firstCar = objects[0];
-  /// int count = objects.length;
-  /// var sortedObjects = objects.sort("make");
-  /// ```
-  Results<T> objects<T extends RealmObject>() {
-    String typeName = _getRealmObjectName<T>();
-    var results = _objects(typeName);  
-    return new Results<T>(results);
-  }
-  RealmResults _objects(String typeName) native "Realm_objects";
   
-  /// Synchronously call the provided callback inside a write transaction. 
-  /// If an exception happens inside a transaction, you'll lose the changes in that transaction, 
-  /// but the Realm itself won't be affected (or corrupted).
-  /// 
-  /// Nesting transactions (calling `write()` within `write()`) will throw an exception.
-  void write(VoidCallback callback) native "Realm_write";
-
-  String _getRealmObjectName<T>() {
-    var schema = TypeStaticProperties.getValue(T, "schema");
-    if (schema == null) {
-      throw new Exception("Class ${T} was not registered in the schema for this Realm");
-    }
-    String name = schema.name;
-    return name;
-  }
-
-  /// Add a [ListenerCallback] callback for the specified event [event].
-  /// 
-  /// The provided callback will be called when write transactions are committed.
-  /// Creating a new write transactions within the callback will throw an exception 
-  /// if there is a current transaction already open
-  /// You can check if there is a current transaction with `realm.IsInTransaction`
-  void addListener(Event event, ListenerCallback callback) {
-    _addListener(event.toString().split(".")[1], callback);
-  }
-  void _addListener(String name, ListenerCallback callback) native 'Realm_addListener';
-
-  /// Remove a [ListenerCallback] callback for the specified event [event].
-  /// 
-  /// The callback argument should be the same callback reference used in a previous call to [addListener]
-  /// ```dart
-  /// var callback = (realm, name) { ... }
-  /// realm.addListener(Event.change, callback);
-  /// realm.removeListener(Event.change, callback);
-  /// ```
-  void removeListener(Event event, ListenerCallback callback) {
-    _removeListener(event.toString().split(".")[1], callback);
-  }
-  void _removeListener(String name, ListenerCallback callback) native 'Realm_removeListener';
-
-  /// Closes this [Realm] so it may be re-opened with a newer schema version. 
-  /// All objects and collections from this Realm are no longer valid after calling this method.
-  /// This method will not throw an exception if called multiple times.
-  close() native 'Realm_close';
+  static String get version => realmCore.libraryVersion;
```

Comment:
Should this return the version of the dart library or the version of Core?

Reply:
None of these. I will be removing it. Good to point it out

Reply:
done